force-app/main/default/classes/extentionController.cls

```diff
@@ -0,0 +1,81 @@
+public class extentionController {
+    
+private Account acct;
+public List<contact> extContacts{get;set;}
+public Map<Integer, Contact> ContactMap {get;set;}
+public String searchText{get
+    {if(searchText == null) searchText = ''; return searchText;}
+        set;}
+
+	//Constructing extension controller for standard account controller
+    public extentionController(ApexPages.StandardController stdCont)
+    {
+        this.acct = (Account)stdCont.getRecord();
+        extContacts = [SELECT id, LastName, Title, Phone, Is_Primary_Contact__c FROM contact WHERE accountid = :acct.id ORDER BY LastName];
```

Comment:
please use this. for all class variables

Reply:
please follow our Best practices - all DMLs and Querries have to be in try/catch blocks